### DISCUSSION TITLE
Improvement suggestion

### DISCUSSION THREAD

**Discussion by abctje** [jonaswinkler/paperless-ng#693] [Feature Requests]
Thank you all for putting in so much work for this great tool! Ik have a view things on my list that I like to share with you.

- put deleted items in a deleted items folder (move) with the possibility to retrieve if needed. Or maybe give a document a "deleted" tag and a button to show deleted tagged items.
- restore tag selection when closing document or saving after edit. At this moment it restores the "view"  in stead of the selected tags.
- Logfile: put the newest items on top of the list to make it easy to see last log entries.

I hope it makes sense, if not I'll try to clarify. Just let me know!

**Comment by jonaswinkler** [comment]
> * put deleted items in a deleted items folder (move) with the possibility to retrieve if needed. Or maybe give a document a "deleted" tag and a button to show deleted tagged items.

Good idea. I'd assume when you talk folders, you'd want an additional folder on disk, i.e. `/usr/src/paperless/media/trash`, and it simply moves deleted documents there.

> * restore tag selection when closing document or saving after edit. At this moment it restores the "view"  in stead of the selected tags.

Hm. When switching between saved views with the sidebar, would you still like the view to reset to its defaults?


> * Logfile: put the newest items on top of the list to make it easy to see last log entries.

I thought about this as well, but the log may also contain detailed error messages ("stack traces"), and these really don't make any sence when printed upside down.

**Reply by abctje** [reply]
*** /usr/src/paperless/media/trash, and it simply moves deleted documents there
Exactly! And maybe make it easy to view the folder content ant if necessary put selected file back into the consume folder?

restore tag:
I select a View and than select a one or more tags within that view
Then edit a document and close it. Paperless goes back to the view
But the tag selection is gone, and you have to select the tags again


*** Log printed upside down
True!
